### DISCUSSION TITLE
manual mesh saving with octopriny

### DISCUSSION THREAD

**Discussion by tallumbroso** [Jyers/Marlin#978] [Q&A] [answered]
Hello, i would like to know if i am using a raspberry pi with octoprint but with your software and do the manual mesh option, where does it save it and how can I make sure that octoprint uses that mesh before every print?

**Comment by Jyers** [comment, marked as answer]
It would be helpful to know what config you're using (ManualMesh vs UBL) but generally as long as you have a M420 S1 in your start gcode it should load and use the mesh.

**Reply by tallumbroso** [reply]
Hey Thanks for the Answer, Im using UBL and saved the mesh on the 0 slot, entered the M420 S1 in my start Gcode command on octoprint. But when i start a new print and look in the Terminal'i never find the M420 command being sent/ so wanted to make sure it is really working.

**Reply by Jyers** [reply]
The best way to be absolutely sure your mesh is being applied is to watch the Z coupler very closely. You should see it move very slightly during large moves across the bed.
